core/auth/auth_type_oidc.go

```diff
@@ -313,6 +313,11 @@ func (a *oidcAuthImpl) loadOidcTokensAndInfo(bodyData map[string]string, oidcCon
 	lastName, _ := userClaims[identityProviderSetting.LastNameField].(string)
 	//email
 	email, _ := userClaims[identityProviderSetting.EmailField].(string)
+	//farpa
+	farpa, ok := userClaims[identityProviderSetting.Farpa].(bool)
```

Comment:
Are you sure that this will not fail if the value is "true" instead of true? Please check and verify that it will not fail.